lerna.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "node_modules/@lerna-lite/cli/schemas/lerna-schema.json",
+  "version": "independent",
+  "npmClient": "yarn",
+  "verbose": true,
+  "packages": [
+    "./packages/*"
+  ],
+  "conventionalCommits": true,
+  "changelog": true,
+  "message": "chore(release): publish",
+  "createRelease": "github",
+  "command": {
+    "publish": {
+      "tag": true,
+      "push": true,
+      "pushArgs": [
+        "--follow-tags"
+      ],
+      "yes": true,
+      "registry": "http://localhost:4873"
```

Comment:
You probably don't need this.

Reply:
Oops, yes, that's very likely :see_no_evil: , removed now